### PR TITLE
Fixup cmake to work better in QtCreator

### PR DESCRIPTION
**Pull request by dpniel**
You should now be able to open the CMakeLists.txt file in qtcreator and see all the project files

### CODE REVIEW

**Comment by dpniel** [comment]
cmake-fixup

In case you don't have qtcreator installed. This is what it should look like now

File: CMakeLists.txt [hunk 1]
```diff
@@ -59,3 +59,38 @@ add_custom_target(${DESKTOP_FILE_NAME} ALL
...
+
+###########################################
+#    Make things useful in QtCreator.     #
+#                                         #
+# !!! No need to change anything here !!! #
+###########################################
+
+
+# List of assets you want visible
+set(CLICK_AND_OTHER_SHMUCK_MANIFESTS
```
Comment (NeoTheThird) [inline CMakeLists.txt]: lol

**Comment by NeoTheThird** [comment]
I'm not really sure, if this is necessary, since you don't edit stuff like the license, but if it's useful to you, it might be useful to others. Thanks for the PR!

File: CMakeLists.txt (hunk 1 above)
Comment (dpniel) [inline CMakeLists.txt]: :-)

**Comment by dpniel** [comment]
I suppose the license file could be dropped. The others I think are useful to have visible though

**Comment by NeoTheThird** [comment]
Yeah, doesn't hurt. I'll just let it in there. Doesn't really matter to me, since i'm using atom (which just uses git to display the file tree) but i guess it's better for qt creator users to have easy access. Thanks :)

**Pull request merged**